docs/SpiraApps/QuickTasks.md

```diff
@@ -0,0 +1,46 @@
+# Quick Tasks SpiraApp
+
+!!! warning "This SpiraApp's functionality requires SpiraTeam or SpiraPlan to work."
+
+This SpiraApp lets users create, complete, and view tasks from a convenient Product Home Page widget. 
+
+## Setup
+To effectively implement Quick Tasks in your product, you have to set up the Task Type that will be used and make it use a specific workflow. 
+
+### Product Template Setup
+You need to configure each product template for the product(s) you want to use Quick Tasks:
+
+1. Go to Tasks > Workflow and add a new Workflow called "Quick Task Workflow". Do not change anything else about this workflow.
+
+2. Go to Tasks > Type and add a new Type called "Quick Task" (or other name of your choice) and select the Workflow you just created
+
+![A row in the Task Type table containing the following: Type ID number, Type name, selected workflow, "No" (for pull request switcher), an empty radio button (for default type), "Yes" (for Active switcher). There is a red outlined box around the Type ID number and the selected workflow](img/quicktasks-task-type-setting.png)
+
+!!! info "Reusing a Task Type"
+    If your team already uses a special Task Type for miscellaneous to-do items, you can use that instead of creating a new one, just make sure to associate it with the Quick Task Workflow you created on step 1.
+
+### Product Settings
+For each product that uses Quick Tasks, a product administrator must select the default Quick Task Type in the Quick Tasks SpiraApp product settings.
```

Comment:
What happens if they do not? Is this definitely required for the app to function?

Reply:
Yes, and there's an error message if nothing was selected